travel_platform/travel_app/static/travel_app/js/foursquare_venues.js

```diff
@@ -0,0 +1,8 @@
+$(function () {
+    $("#foursquare-city").submit(function (e) {
```

Comment:
This code looks like a duplication of the same part of code from `attractions.js`.
It would be great to have some `common.js` or similar for storing common functionality.